packages/metal3/charts/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+appVersion: 1.16.0
+dependencies:
+- alias: metal3-baremetal-operator
+  name: baremetal-operator
+  repository: file://./charts/baremetal-operator
+  version: 0.1.1
+- condition: ironic.enabled
+  name: ironic
```

Comment:
We're missing the previous aliases e.g metal3-ironic etc - I think that's why the global values override isn't working (the values file still references the alias names).

To avoid unnecessarily breaking existing user configs should we just restore the aliases?